recipes/vscode-jupyter/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "vscode-jupyter" %}
+{% set version = "2020.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/microsoft/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 99502db43f9f2ac4185d55ae51a70c81ffe9353983dcaca5cd1a7cf29f783a11
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - nodejs 12
+    - code-server >=3.9
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - nodejs 12
+    - code-server >=3.9
+  run:
+    - code-server >=3.9
+    - python-jsonrpc-server 0.4.0
+    - python >=2.7
```

The linter will like this
```suggestion
    - python *
```